README.md

```
@@ -1 +1,93 @@
+### 运行
+
+```bash
+poetry run python -m hybrid_a_star_planner.main
+```
+
+如在无 GUI 环境运行，建议将可视化改为保存图片，以避免
+`plt.show()` 阻塞或后端警告。
+
+## 项目结构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)